graphing/src/cloudevolution/views.py

```diff
@@ -94,7 +94,54 @@ def expt_name(request, experiment):
 
 	return render(request, "experiment.html", context)
 
+def dilutions(request, experiment):
+	sidebar_links, subdir_log = file_scan('expt')
+	vial_count = range(0, 16)
+	expt_dir, expt_subdir = file_scan(experiment)
+	rootdir = os.path.dirname(os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__)))))
+	evolver_dir = rootdir + '/experiment'
+	pump_cal = evolver_dir + "/%s/pump_cal.txt" % (expt_subdir[0])
+
+	cal = np.genfromtxt(pump_cal, delimiter="\t")
+	diluted = []
+	efficiency = []
+	last = []
+
+	for vial in vial_count:
+		pump_dir = evolver_dir + "/%s/%s/pump_log/vial%s_pump_log.txt" % (expt_subdir[0], experiment, vial)
```

Comment:
Can you use `os.path.join` instead here and in any other cases where you are generating file paths? Needs to cross-platform compatible.

Reply:
Done. Modified all of them, including those that existed before this feature.